src/components/ChatHome/ChatHome.tsx

```diff
@@ -143,7 +162,10 @@ const Prompts = () => {
         prompt={{slug: 'create', description: 'Create your own chat app'}}
         icon={<AddCircle variant={"Bold"} color={variables.light60}/>}
         type={'default'}
-        onClick={() => openModal('create-prompt')}
+        onClick={() => {
+          openModal('create-prompt')
+          trackEvent('hl_chat_create_prompt_from_list', {});
```

Comment:
This could be `trackEvent('Chat Start Create Chat App', { context: 'Chat Home List' })`, and wherever else you create chat apps would change the `context` property.

Reply:
Done